### PR TITLE
fix(FEC-13054): Top bar missing when using YouTube entries

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves FEC-13054

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/top-bar/_top-bar.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
   width: 100%;
   top: 0;
   left: 0;
+  z-index: 1;
```
Comment (semarche-kaltura) [inline src/components/top-bar/_top-bar.scss:15]: why the changes was applied in previous PR?

**Review by semarche-kaltura — APPROVED** [review]

File: src/components/top-bar/_top-bar.scss (hunk 1 above)
Comment (JonathanTGold) [reply src/components/top-bar/_top-bar.scss:15]: According to the last (current) implementation made at the end it actually wasn't needed
